scripts/script_import_test.py

```diff
@@ -30,32 +30,39 @@
 
 from core.tests import test_utils
 
+from typing import Any, Dict, List, Tuple
+from typing_extensions import Literal
+
 
 class InstallThirdPartyLibsImportTests(test_utils.GenericTestBase):
     """Tests import of install third party libs."""
 
-    def setUp(self):
+    def setUp(self) -> None:
         super(InstallThirdPartyLibsImportTests, self).setUp()
-        self.commands = []
-        def mock_popen_error_call(unused_cmd_tokens, *args, **kwargs): # pylint: disable=unused-argument
-            class Ret(test_utils.GenericTestBase):
-                """Return object that gives user-prefix error."""
+        self.commands: List[str] = []
+        class Ret(test_utils.GenericTestBase):
+            """Return object that gives user-prefix error."""
 
-                def __init__(self):  # pylint: disable=super-init-not-called
-                    self.returncode = 1
-                def communicate(self):
-                    """Return user-prefix error as stderr."""
-                    return b'', b'can\'t combine user with prefix'
+            def __init__(self) -> None:  # pylint: disable=super-init-not-called
+                self.returncode = 1
+            def communicate(self) -> Tuple[bytes, bytes]:
+                """Return user-prefix error as stderr."""
+                return b'', b'can\'t combine user with prefix'
+        def mock_popen_error_call(
+            unused_cmd_tokens: List[str], # pylint: disable=unused-argument
+            *args: Tuple[Any], # pylint: disable=unused-argument
+            **kwargs: Dict[str, Any] # pylint: disable=unused-argument
+        ) -> Ret:
             return Ret()
-        def mock_check_call(cmd_tokens):
+        def mock_check_call(cmd_tokens: List[str]) -> None:
```

Comment:
```suggestion

        def mock_check_call(cmd_tokens: List[str]) -> None:
```

Reply:
If we try to add this change, lint checks will give this error
### scripts/script_import_test.py:60:9: E303 too many blank lines (2).

Reply:
Weird this seems a bit wrong. There is only one line, right?

Reply:
Oh, the empty line is there actually.